parsl/dataflow/dflow.py

```diff
@@ -76,6 +85,24 @@ def __init__(self, config=Config()):
         self.usage_tracker = UsageTracker(self)
         self.usage_tracker.send_message()
 
+        # ES logging
+        self.db_logger_config = db_logger_config
+        self.db_logger = get_db_logger(enable_es_logging=False) if self.db_logger_config is None else get_db_logger(**self.db_logger_config)
+        self.workflow_name = str(inspect.stack()[1][1])
+        self.time_began = datetime.now()
+        self.time_completed = None
+        self.run_id = self.workflow_name + "-" + str(self.time_began.minute)
+        self.dashboard = "https://search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-east-1.es.amazonaws.com/_plugin/kibana/app/kibana#/dashboard/63549e70-757a-11e8-90be-3b2ab570ca29?_g=(refreshInterval%3A('%24%24hashKey'%3A'object%3A497'%2Cdisplay%3A'10%20seconds'%2Cpause%3A!f%2Csection%3A1%2Cvalue%3A10000)%2Ctime%3A(from%3Anow-15m%2Cinterval%3A'1m'%2Cmode%3Aquick%2Ctimezone%3AAmerica%2FChicago%2Cto%3Anow))"  # noqa: E501
```

Comment:
I think this should be configurable (part of the `db_logger_config`), though I support using this address as the default for now.

Reply:
Would it be okay if the db_logger_config simply took a "dashboard_link" keyword arg?